pages/profile/read/views/index.jsx

```diff
@@ -50,6 +50,20 @@ const Index = ({
           <Profile profile={model} establishment={establishment} allowedActions={allowedActions} isOwnProfile={isOwnProfile} />
         </div>
       </div>
+      {
+        allowedActions.includes('training.read') && (
+          <section className="profile-section">
+            <h3><Snippet>training</Snippet></h3>
+            {
+              !!certificates.length && <TrainingSummary certificates={certificates} />
```

Comment:
It feels like this check would be better done inside `<TrainingSummary />`

i.e.

```
if (!certificates || !certificates.length) {
  return null;
}
```

Reply:
It would negate the need to default the certificates here as well.

Reply:
it already is, I just need to remove the check here